From: Jan Engelhardt <jengelh@inai.de>
Date: 2021-08-23 14:47:30.341142824 +0200
References: https://bugzilla.opensuse.org/show_bug.cgi?id=1189086
References: https://bugzilla.redhat.com/show_bug.cgi?id=1984776

glibc-2.34 added closefrom. Rename fuse functions to work around the collision.
---
 util/ulockmgr_server.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

Index: fuse-2.9.9/util/ulockmgr_server.c
===================================================================
--- fuse-2.9.9.orig/util/ulockmgr_server.c
+++ fuse-2.9.9/util/ulockmgr_server.c
@@ -124,7 +124,7 @@ static int receive_message(int sock, voi
 	return res;
 }
 
-static int closefrom(int minfd)
+static int fuse_closefrom(int minfd)
 {
 	DIR *dir = opendir("/proc/self/fd");
 	if (dir) {
@@ -384,7 +384,7 @@ int main(int argc, char *argv[])
 		dup2(nullfd, 1);
 	}
 	close(3);
-	closefrom(5);
+	fuse_closefrom(5);
 	while (1) {
 		char c;
 		int sock;
